playbooks/bootstrap-cluster-custom-ssh.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+- name: Test 'cephadm_bootstrap' module
+  hosts: admin
+  gather_facts: false
+  become: true
+  any_errors_fatal: true
+
+  tasks:
+    - name: Bootstrap with dashboard set to `true` without credenatials
+      cephadm_bootstrap:
+        mon_ip: "{{ mon_ip }}"
+        dashboard: true
+        monitoring: false
+        firewalld: false
+        ssh_user: cephuser
```

Comment:
Will user `cephuser` available in cluster ?

Reply:
Yes Vaibhav, its available in https://github.com/red-hat-storage/cephci/blob/master/conf/inventory/rhel-8.7-server-x86_64.yaml#L15

Reply:
+1